man/m_add_cylinder.Rd

```diff
@@ -90,7 +90,7 @@ r3dmol() \%>\%
   ) \%>\%
   m_add_res_labels(m_sel(resi = 1:4, chain = "A"))
 
-  # The same scene achieved with m_multi_resi_sel()
+# The same scene achieved with m_multi_resi_sel()
```

Comment:
You could add a commit to bring the master into branch to remove not related changes.

Reply:
I am unsure how to do this. I'm still learning how to use git collaboratively. Apologies!

Reply:
That's OK, I will fix it~
And here is a links for exmaple: https://stackoverflow.com/a/5340774/8904293

Reply:
Sorry, it's my fault, the code in master is not formatted so your PR is correct.